lib/GTK/Simple/NativeLib.pm6

```diff
@@ -92,18 +92,11 @@ sub gobject-lib is export {
 
 sub find-bundled($lib is copy) {
     # if we can't find one, assume there's a system install
-    my $base = "lib/GTK/$lib";
-    for @*INC {
-        if my @files = ($_.files($base) || $_.files("blib/$base")) {
-            my $files = @files[0]<files>;
-            my $tmp = $files{$base} || $files{"blib/$base"};
+    my $base = "blib/lib/GTK/$lib";
 
-            # copy to a temp dir
-            $tmp.IO.copy($*SPEC.tmpdir ~ '\\' ~ $lib);
+    if my $file = %?RESOURCES{$base} {
+            $file.IO.copy($*SPEC.tmpdir ~ '\\' ~ $lib);
```

Comment:
I have to refer to this commit: 702cd066a856a810896ba9a57628e84f4bbf3d61.
I left the copy operation in, with this commit I primarily tried to get GTK::Simple to work on Windows again.